examples/image-to-text/run_pipeline.py

```diff
@@ -325,6 +325,11 @@ def main():
             ]
             args.prompt = processor.apply_chat_template(conversation, add_generation_prompt=True)
 
+    if model_type == "mllama" and args.use_flash_attention:
```

Comment:
why not just allow user to select attn_implementation + add readme section about it?